lib/utils/parseSelector.mjs

```diff
@@ -4,12 +4,20 @@ import selectorParser from 'postcss-selector-parser';
  * @param {string} selector
  * @param {import('stylelint').PostcssResult} result
  * @param {import('postcss').Node} node
- * @param {(root: import('postcss-selector-parser').Root) => void} callback
- * @returns {string | undefined}
+ * @param {(root: import('postcss-selector-parser').Root) => void} [callback] - Deprecated. It will be removed in the future.
```

Comment:
minor nit

https://github.com/jsdoc/jsdoc.github.io/issues/39#issuecomment-18919123

Reply:
I feel that not using `@deprecated` is more explicit though.